superset/commands/report/exceptions.py

```diff
@@ -93,6 +93,34 @@ def __init__(self) -> None:
         )
 
 
+class ReportScheduleFrequencyNotAllowed(ValidationError):
+    """
+    Marshmallow validation error for report schedule configured to run more
+    frequently than allowed
+    """
+
+    def __init__(
+        self,
+        report_type: str = "Report",
+        minimum_interval: int = 120,
+    ) -> None:
+        interval_in_minutes = (
+            minimum_interval / 60
+            if not minimum_interval % 60
+            else minimum_interval // 60 + 1
+        )
```

Comment:
I think this does it:

```suggestion
        interval_in_minutes = math.ceil(minimum_interval / 60)
```

(you need to import `math` from the stdlib, though)